lib/elixir_tic_tac_toe_basic/player.ex

```diff
@@ -0,0 +1,19 @@
+defmodule ElixirTicTacToeBasic.Player do
+  @moduledoc """
+  This module provides functions for making moves in the game.
+  """
+
+  def move(state) do
+    state
+    |> ElixirTicTacToeBasic.Prompter.get_input()
```

Comment:
**question (non-blocking)**: Can you talk about how SOLID principles apply here and the trade-offs you encountered when introducing this dependency inside `Player`?